.github/patches/extensions/sqlite_scanner/wrap_idx_t.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/src/storage/sqlite_table_entry.cpp b/src/storage/sqlite_table_entry.cpp
+index 9d83f00..0dbbb88 100644
+--- a/src/storage/sqlite_table_entry.cpp
++++ b/src/storage/sqlite_table_entry.cpp
+@@ -53,9 +53,23 @@ TableStorageInfo SQLiteTableEntry::GetStorageInfo(ClientContext &context) {
+ 	auto &transaction = Transaction::Get(context, catalog).Cast<SQLiteTransaction>();
+ 	auto &db = transaction.GetDB();
+ 	TableStorageInfo result;
+-	if (!db.GetMaxRowId(name, result.cardinality)) {
++
++	// Unwrap result.cardinality into idx_t
++	idx_t index = DConstants::INVALID_INDEX;
++	if (result.cardinality.IsValid()) {
```

Comment:
This code looks a bit off to me - `db.GetMaxRowId` sets the index, so this unwrapping here is unnecessary (in fact I think result.cardinality will always be invalid when it is passed in). Something like this makes more sense:

```cpp
idx_t cardinality;
if (!db.GetMaxRowId(name, cardinality)) {
    cardinality = 10000;
}
result.cardinality = cardinality;

```

Reply:
Thanks. I had not looked at the GetMaxRowId code and assumed the parameter was potentially used in the function, while it's useful only for passing the second return value.

Updated with your suggestion, see https://github.com/duckdb/duckdb/pull/11486/files#diff-28770df5881551f236a035166e4743480413d2f219c147c9ee192c8aa5a06e5f